types/lib/query-interface.d.ts

```diff
@@ -630,21 +630,20 @@ export class QueryInterface {
   ): Promise<void>;
 
   /**
-   * Escape an identifier (e.g. a table or attribute name). If force is true, the identifier will be quoted
-   * even if the `quoteIdentifiers` option is false.
+   * Escape a table name
    */
-  public quoteIdentifier(identifier: string, force: boolean): string;
+  public quoteTable(identifier: TableName): string;
```

Comment:
I swapped the order of the functions from

quoteIdentifier
quoteTable
quoteIdentifiers

to

quoteTable
quoteIdentifier
quoteIdentifiers

Hence the diff looks a bit odd